tsl/test/expected/continuous_aggs.out

```diff
@@ -862,10 +868,16 @@ group by time_bucket('1day', timec), location, humidity, temperature;
 NOTICE:  adding index _materialized_hypertable_20_grp_5_5_timec_idx ON _timescaledb_internal._materialized_hypertable_20 USING BTREE(grp_5_5, timec)
 NOTICE:  adding index _materialized_hypertable_20_grp_6_6_timec_idx ON _timescaledb_internal._materialized_hypertable_20 USING BTREE(grp_6_6, timec)
 NOTICE:  adding index _materialized_hypertable_20_grp_7_7_timec_idx ON _timescaledb_internal._materialized_hypertable_20 USING BTREE(grp_7_7, timec)
+SELECT add_refresh_continuous_aggregate_policy('mat_with_test', NULL, '5 h'::interval, '12 h'::interval); 
+ add_refresh_continuous_aggregate_policy 
+-----------------------------------------
+                                    1001
+(1 row)
+
 SELECT alter_job(id, schedule_interval => '1h') FROM _timescaledb_config.bgw_job;
-                                     alter_job                                     
------------------------------------------------------------------------------------
- (1012,"@ 1 hour","@ 0",-1,"@ 48 hours",t,"{""mat_hypertable_id"": 20}",-infinity)
+                                                     alter_job                                                      
+--------------------------------------------------------------------------------------------------------------------
+ (1001,"@ 1 hour","@ 0",-1,"@ 12 hours",t,"{""end_interval"": ""@ 5 hours"", ""mat_hypertable_id"": 20}",-infinity)
```

Comment:
In line with other comment on adding default config: Here it would be useful to see what the `start_interval` is, but since it isn't added to the config, one cannot know without having detailed knowledge of what config keys actually exist.

Reply:
added.